### PR TITLE
created blockchain.legacy.test and blockchain.blobk.legacy.test

### PR DESCRIPTION
**Pull request by SebastianDremo**

### CODE REVIEW

File: src/Nethermind/Ethereum.Blockchain.Block.Legacy.Test/BlockGasLimitTests.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    [Parallelizable(ParallelScope.All)]
+    public class BlockGasLimitTest : LegacyBlockchainTestBase
+    {
+        [TestCaseSource(nameof(LoadTests))]
+        public async Task Test(LegacyBlockchainTest test)
+        {
+            await RunTest(test);
+        }
+
+        public static IEnumerable<LegacyBlockchainTest> LoadTests() { return new DirectoryTestsSource("bcBlockGasLimitTest").LoadLegacyTests(); }
```
Comment (LukaszRozmej) [inline src/Nethermind/Ethereum.Blockchain.Block.Legacy.Test/BlockGasLimitTests.cs:18]: Those classes are basically the same with only different directory? Can we remove this duplication?

Comment (SebastianDremo) [reply src/Nethermind/Ethereum.Blockchain.Block.Legacy.Test/BlockGasLimitTests.cs:18]: this the main reason why PR is still under WIP,
my next goal is to make the older ones to get up to date tests

Comment (SebastianDremo) [reply src/Nethermind/Ethereum.Blockchain.Block.Legacy.Test/BlockGasLimitTests.cs:18]: just wanted to show progress tbh and let Tomek run those tests in Blockchain.Legacy.Test

Comment (LukaszRozmej) [reply src/Nethermind/Ethereum.Blockchain.Block.Legacy.Test/BlockGasLimitTests.cs:18]: ok, use Draft PR in future.

**Pull request merged**